src/components/modules/epics/ContributionsEpicReminder.tsx

```diff
@@ -119,6 +120,18 @@ type ReminderPayload = {
     reminderDate: string;
 };
 
+const dateDiff = (start: Date, end: Date): number => {
+    const twentyFourHours = 86400000;
+    return Math.round(Math.abs((start.valueOf() - end.valueOf()) / twentyFourHours));
```

Comment:
can be `Math.round((end.valueOf() - start.valueOf()) / twentyFourHours)`

Reply:
👍